tests/ui/test_notifications.py

```diff
@@ -78,7 +78,7 @@ def app():
 
     page.click('.bk-btn')
 
-    expect(page.locator('.MuiAlert-message')).to_have_count(2)
+    wait_until(lambda: expect(page.locator('.MuiAlert-message')).to_have_count(2), page)
```

Comment:
Thanks for trying to fix, that said this pattern does not really make sense and all this change actually does is ignore this step.